### PR TITLE
Polyswarm bug fix

### PR DESCRIPTION
**Pull request by reutshal**
## Status
- [] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23110

## Description
- Added the 'file' argument to the 'file' command as default.
- The 'hash' argument is no longer mandatory
 - The validation fails due to some context problems which will be solved in the contributor's enhancement in https://github.com/demisto/etc/issues/20031
## Does it break backward compatibility?
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation 
- [X] Code Review

### CODE REVIEW

File: Integrations/PolySwarm/PolySwarm.py [hunk 1]
```diff
@@ -269,7 +269,10 @@ def test_connectivity(self):
         return True
 
     def file_reputation(self, param):
-        title = 'PolySwarm File Reputation for Hash: %s' % param['hash']
+        file_hash = param.get('hash', param.get('file'))
+        if not file_hash:
+            return_error("Please specify a hash or a file to enrich.")
```
Comment (ronykoz) [inline Integrations/PolySwarm/PolySwarm.py:274]: Specify a file hash to enrich

File: Integrations/PolySwarm/PolySwarm.py [hunk 2]
```diff
@@ -269,7 +269,10 @@ def test_connectivity(self):
         return True
 
     def file_reputation(self, param):
-        title = 'PolySwarm File Reputation for Hash: %s' % param['hash']
+        file_hash = param.get('hash', param.get('file'))
```
Comment (teizenman) [inline Integrations/PolySwarm/PolySwarm.py:272]: I think its better to switch it

```suggestion
        file_hash = param.get('file', param.get('hash'))
```

**Comment by kirbles19** [comment]
@reutshal Done.

**Comment by ronykoz** [comment]
Force merging because the contributor will add the missing fields In a new PR

**Pull request merged**